### PR TITLE
feat(route): add 豆瓣当月期待剧集/电影推荐

### PR DESCRIPTION
**Pull request by honue**
## Involved Issue / 该 PR 相关 Issue

Close #

## Example for the Proposed Route(s) / 路由地址示例

```routes
/douban/recommended/movie
/douban/recommended/tv/score=7
```

## New RSS Route Checklist / 新 RSS 路由检查表
  
- [x] New Route / 新的路由
  - [ ] Follows [v2 Script Standard](https://docs.rsshub.app/joinus/advanced/script-standard) / 跟随 [v2 路由规范](https://docs.rsshub.app/zh/joinus/advanced/script-standard)
- [x] Documentation / 文档说明
  - [ ] EN / 英文文档
  - [x] CN / 中文文档
- [ ] Full text / 全文获取
  - [ ] Use cache / 使用缓存
- [ ] Anti-bot or rate limit / 反爬/频率限制 
  - [ ] If yes, do your code reflect this sign? / 如果有, 是否有对应的措施? 
- [ ] [Date and time](https://docs.rsshub.app/joinus/advanced/pub-date) / [日期和时间](https://docs.rsshub.app/zh/joinus/advanced/pub-date)
  - [ ] Parsed / 可以解析
  - [ ] Correct time zone / 时区正确
- [ ] New package added / 添加了新的包
- [ ] `Puppeteer`

### CODE REVIEW

File: lib/v2/douban/other/recommended.js [hunk 1]
```diff
@@ -0,0 +1,74 @@
...
+module.exports = async (ctx) => {
+    const subject_type = ctx.params.type || 'tv';
+    const apikey = '0ac44ae016490db2204ce0a042db2916';
+    let url = `https://frodo.douban.com/api/v2/skynet/new_playlists?apikey=${apikey}&subject_type=${subject_type}`;
+    let response = await got({
+        method: 'get',
+        url,
+        headers: {
+            'User-Agent': `MicroMessenger/`,
+            Referer: `https://servicewechat.com/wx2f9b06c1de1ccfca/91/page-frame.html`,
```
Comment (TonyRL) [inline lib/v2/douban/other/recommended.js:15]:
```suggestion
            'User-Agent': 'MicroMessenger/',
            Referer: 'https://servicewechat.com/wx2f9b06c1de1ccfca/91/page-frame.html',
```

File: lib/v2/douban/other/recommended.js [hunk 2]
```diff
@@ -0,0 +1,74 @@
+const got = require('@/utils/got');
+const path = require('path');
+const { art } = require('@/utils/render');
+const { fallback, queryToInteger } = require('@/utils/readable-social');
+
+module.exports = async (ctx) => {
+    const subject_type = ctx.params.type || 'tv';
```
Comment (TonyRL) [inline lib/v2/douban/other/recommended.js:7]: `camelCase`

File: lib/v2/douban/other/recommended.js [hunk 3]
```diff
@@ -0,0 +1,74 @@
...
+    const year = date.getFullYear();
+    const mon = date.getMonth() + 1;
+    let items = response.data.data[0].items;
+
+    for (const item of items) {
+        if (item.title.startsWith(`${year}年${mon}月`)) {
+            subject_collection_id = item.id;
+            break;
+        }
+    }
```
Comment (TonyRL) [inline lib/v2/douban/other/recommended.js:30]:
```suggestion
    subject_collection_id = items.find((item) => item.title.startsWith(`${year}年${mon}月`)).id;
```

File: lib/v2/douban/other/recommended.js (hunk 2 above)
Comment (honue) [reply lib/v2/douban/other/recommended.js]: done

**Pull request merged**